### ISSUE TITLE
Single-item URLs should have form: ursus.library.ucla.edu/ark:/{{ark/identifier}}

### ISSUE THREAD

**Issue by andrewbenedictwallace** [UCLALibrary/ursus#175]
In conformance with [the ark specification](https://escholarship.org/uc/item/9p9863nc)

depends on #170
child of github.com/UCLALibrary/californica/issues/336

**Comment by jendiamond** [comment]
```
   http://example.org/ark:/12025/654xz321/s3/f8.05v.tiff    \________________/ \__/ \___/ \______/ \____________/      (replaceable)     |     |      |       Qualifier           |       ARK Label  |      |    (NMA-supported)           |                  |      | Name Mapping Authority       |    Name (NAA-assigned)    Hostport (NMAH)           |                     Name Assigning Authority Number (NAAN)

The ARK syntax can be summarized,  where the NMAH and Qualifier parts are in brackets to indicate that they are optional.
```

**Comment by jendiamond** [comment]
re-index all the Work records

**Comment by jendiamond** [comment]
- [ ] figure out how to use ARK-based URLs instead of ID-based URLs.

**Comment by mark-dce** [comment]
I'm blocking this ticket until we can have a conversation about requirements around 
* Resolvers like n2t.org, ark.cdlib.org
* Where and how external references get updated
* Whether the authoritative citation link has https://ursus.library.ucla.edu/ in it or whether we actually provide a resolver link in authoritative citations

**Comment by mark-dce** [comment]
Superseded by requirements express in #270 and children.
